plugins/plugin-postcss/README.md

```diff
@@ -1,13 +1,13 @@
 # @snowpack/plugin-postcss
 
-Run [PostCSS](https://github.com/postcss/postcss) on all `.css` files, including ones generated from Sass, Vue, and Svelte.
+Runs [PostCSS](https://github.com/postcss/postcss) on all `.css` files, including ones generated from Sass, Vue, and Svelte.
 
 ### Usage
 
-From a terminal, run the following:
+Install @snowpack/plugin-postcss, PostCSS, and your PostCSS plugins (not shown):
 
 ```
-npm install --save-dev @snowpack/plugin-postcss postcss
+npm i -D @snowpack/plugin-postcss postcss
```

Comment:
I’d prefer the original, verbose syntax as it’s friendlier to newcomers (if you are familiar with the shortcuts you’ll use them)

Reply:
If that's the preference then I'll also revert the install in the @web/test-runner guide.